CONTRIBUTING.md

```diff
@@ -581,18 +585,67 @@ mod tests {
 }
 ```
 
-For more detailed guidance on how to add Rust testing you can refer to the Rust
+For more detailed guidance on how to write Rust tests, you can refer to the Rust
 documentation's [guide on writing tests](https://doc.rust-lang.org/book/ch11-01-writing-tests.html).
 
-To run the Rust-space tests, do
+Rust tests are run separately from the Python tests. The easiest way to run
+them is via ``tox``, which creates an isolated venv and pre-installs ``qiskit``
+prior to running ``cargo test``:
+
+```bash
+tox -erust
+```
+
+You can also execute them directly in your own virtual environment with these
+commands (which is what the ``tox`` env is doing under the hood):
 
 ```bash
-cargo test --no-default-features
+python setup.py build_rust --release --inplace
```

Comment:
I'd actually change this to not include `--release` by default. That just increases the build time and we shouldn't be bottlenecked on the python extension's execution time in rust tests.

Reply:
Should be done in 234b6a56e847914cfb75c6462489bb1900bb4ea1 as well.